### PR TITLE
feat: wallet-service rollout proof-of-concept with unleash

### PR DESCRIPTION
**Pull request by andreabadesso**

### CODE REVIEW

File: src/featureFlags.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    const client = new UnleashClient({
+      url: UNLEASH_URL,
+      clientKey: UNLEASH_CLIENT_KEY,
+      refreshInterval: 15,
+      appName: `wallet-service-mobile-${Platform.OS}`,
+    });
+
+    client.updateContext({ userId });
+
+    await client.fetchToggles();
```
[the remark this answers is not in the document]
Comment (andreabadesso) [reply src/featureFlags.js:28]: I've just checked and I believe we are safe even with the request failing silently

`isEnabled` will return `false` by default if the toggle is not present

File: ios/Podfile.lock [hunk 2]
```diff
@@ -117,4 +117,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: be40117dd8d8b213dbec64f11efa69f4de1195d0
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.10.2
```
[the remark this answers is not in the document]
Comment (andreabadesso) [reply ios/Podfile.lock:120]: On pod install, cocoapods will write to the `Podfile.lock` file the current installed version of cocoapods, in my case it is the latest one:

```
➜  ios git:(feat/unleash) ✗ gem which cocoapods
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.10.2/lib/cocoapods.rb
```

I don't see any problems with that, but if you do, I can downgrade my cocoapods.

What do you think?

[the remark this answers is not in the document]
Comment (pedroferreira1) [reply ios/Podfile.lock:120]: It's okay, we just need to test all the libraries used by cocoapods.

**Review by pedroferreira1 — APPROVED** [review]

**Comment by luislhl** [comment]
Why do we have `yarn.lock` and `package-lock.json` together?

File: src/reducer.js [hunk 3]
```diff
@@ -468,6 +470,11 @@ const onSetWallet = (state, action) => {
   };
 };
 
+const onSetUseWalletService = (state, action) => ({
+  ...state,
+  useWalletService: action.payload,
```
Comment (luislhl) [inline src/reducer.js:475]: We are not using this state anywhere in the application yet, right?

**Review by luislhl — APPROVED** [review]

File: src/reducer.js (hunk 3 above)
Comment (andreabadesso) [reply src/reducer.js:475]: We are using it, it is loaded from the props on the following screens:

```
src/screens/CreateTokenConfirm.js
src/screens/SendConfirmScreen.js
src/screens/LoadHistoryScreen.js
```

**Pull request merged**